src/ws/cockpitwebservice.c

```diff
@@ -457,11 +457,6 @@ process_socket_authorize (CockpitWebService *self,
   cockpit_memory_clear (data, length);
 
 out:
-  if (password)
-    {
-      cockpit_memory_clear (password, -1);
-      free (password);
-    }
```

Comment:
Oh, I see! I misinterpreted the commit log, I thought we weren't using the `password` here at all any more due to the authorize rework.